test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -251,10 +251,13 @@ def send_keys(self, element, text, clear=True, ctrla=False):
             self.execute_script('var ev = new Event("change", { bubbles: true, cancelable: false }); arguments[0].dispatchEvent(ev);', element)
             self.execute_script('var ev = new Event("change", { bubbles: true, cancelable: false }); arguments[0].dispatchEvent(ev);', element)
 
-    def check_box(self, element, checked=True):
+    def check_box(self, element, checked=True, retry=5):
```

Comment:
I don't see why the checkbox will take longer than one try for it's state to change.
If it's really taking long - it's a problem with app itself and we should fix it.

Reply:
I have encountered some situations which may need this.For example, in some machines which have high latency, I can not check some boxes which are in some confirmation dialog if I only check once.If I check second time or add a little delay,  I can check these boxes sometimes.By manual, I don't encounter the situations which is checkbox need click twice. So I add this here.

Reply:
Generally though such 'hacks' can hide real problems and are highly discouraged. Please let's leave without the hack and if we really encounter the test flake in CI often let's reconsider it.

Reply:
OK